cmd/lncli/commands.go

```diff
@@ -2183,9 +2189,11 @@ func updateChannelPolicy(ctx *cli.Context) error {
 	}
 
 	req := &lnrpc.PolicyUpdateRequest{
-		BaseFeeMsat:   baseFee,
-		TimeLockDelta: uint32(timeLockDelta),
-		MaxHtlcMsat:   ctx.Uint64("max_htlc_msat"),
+		BaseFeeMsat:        baseFee,
+		TimeLockDelta:      uint32(timeLockDelta),
+		MaxHtlcMsat:        ctx.Uint64("max_htlc_msat"),
+		InboundBaseFeeMsat: int32(ctx.Int64("inbound_base_fee_msat")),
```

Comment:
Q: We do not care about a potential overflow because fees should never be that high, or what happens if the fees are greater than 32 bits ?

Reply:
A check is a good idea, added.